src/components/molecules/ResizedBadgePreviewsList/ResizedBadgePreviewsList.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, styled } from '@mui/material'
+import React from 'react'
+import './resizedBadgePreviewsList.scss'
+
+export interface ResizedBadgePreviewsListProps {
+  badges: React.ReactNode[]
+}
+
+const defaultValuesForResizedBadgePreviewsListProps = {
+  badges: [],
+}
+
+const ResizedBadgePreviewsListBox = styled(Box)<{ scale?: number }>(({ theme, scale }) => ({
+  padding: theme.spacing(2),
+  scale: `${scale || 1}`,
+  '& :focus': {
+    outline: 'none !important',
+    boxShadow: 'none',
+  },
+}))
+
+export const ResizedBadgePreviewsList = (
+  props: ResizedBadgePreviewsListProps = defaultValuesForResizedBadgePreviewsListProps
+) => {
+  return (
+    <ResizedBadgePreviewsListBox className={'resizedBadgePreviewsList'}>
+      {/* show only the first 5 elements */}
+      {props.badges.slice(0, 5).map((badge, index) => {
```

Comment:
Could it be a parameter? I mean maybe in a range, like 3 to 6 or something that makes sense or "easy" to make work

Reply:
The problem is that I only see designs for the case of having 5. I don't know how it should look like with more or less

Reply:
Make sense, we can do it later if it needed